### PR TITLE
Removing the mention of the deprecated TrueCharts service in TrueNAS SCALE's guide section. 

### PR DESCRIPTION
**Pull request by maru801**
The current mention of TrueNAS on the installation guide mentions a service (TrueCharts) that is being deprecated from TrueNAS.

Source of info: https://truecharts.org/news/scale-deprecation/

The point of the installation guide is meant for installing Jellyfin onto the system. But going forward, TrueCharts will most likely be redirecting users to deploy Jellyfin on another OS and using TrueNAS as just a storage medium for their services. It seems like if TrueCharts is going to remain mentioned in the installation guide, a new entry should be made for it, and the current TrueNAS entry should be separated from it into its own thing.

This pull request:
1. Removes the existing TrueNAS SCALE install mention using TrueCharts from the bottom of the Container guide.
2. Adds a dedicated TrueNAS SCALE page in the install section.
3. Provides two methods of installing Jellyfin directly onto a TrueNAS system.

Furthermore, this pull request will only remain relevant until sometime in Q4 of 2024 as [iXsystems is phasing out Kubernetes in favor of Docker](https://forums.truenas.com/t/the-future-of-electric-eel-and-apps/5409) for Scale's apps. The guide will have to be remade then.

### CODE REVIEW

File: docs/general/installation/truenas-scale.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        - Seeking help for TrueNAS SCALE on the [Jellyfin Forums](https://forum.jellyfin.org/) will depened on active forum members having experience with SCALE.
+    - TrueNAS SCALE is a NAS OS built on Debian that provides an interface to deploy apps. TrueNAS is provided and maintained by [iXsystems](https://www.ixsystems.com/).
+        - Up until SCALE v24.04 (Dragonfish), Kubernetes was used to deploy apps.
+        - According to [an iXsystems staff member's post](https://forums.truenas.com/t/the-future-of-electric-eel-and-apps/5409), starting with SCALE v24.10 (Electric Eel), Kubernetes will be replaced with Docker to deploy apps on SCALE. You'll be able to use Docker Compose on SCALE. The staff member states for those upgrading their SCALE system to v24.10, any installed community apps (only) will be auto-converted to Docker Compose format. This means custom and 3rd party app deployments on SCALE may need user intervention to reconfigure moving forward. Refer to the container installation guide for further assistance with Docker when the time comes.
+        - The steps outlined below apply to SCALE v24.04.
```
Comment (nielsvanvelzen) [inline docs/general/installation/truenas-scale.md:17]: Why use a code block for this part?

Comment (maru801) [reply docs/general/installation/truenas-scale.md:17]: That was unintentional. I was simply visually formatting by eye and wanted a bullet-point list to be seen as belonging to the above "Notes" section. I'm new to markdown, so I wasn't aware a couple of spaces also counted as a code block (just looked it up). I thought backticks or tildes were the only way to make a code block.

You can go ahead and remove the spaces or wait for me to update the pull request later today.

**Comment by felix920506** [comment]
I think adding a Truenas guide and removing truecharts should be separate PRs.

File: docs/general/installation/truenas-scale.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+description: Install on TrueNAS SCALE.
+sidebar_position: 5
+---
+
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.com/truenas-scale/) community app. The source code for the app is available on the TrueNAS [GitHub](https://github.com/truenas/charts/tree/master/community/jellyfin).
+
+### Notes
+
+- For setup/install/network questions, try the [TrueNAS Forums](https://forums.truenas.com/).
+  - Seeking help for TrueNAS SCALE on the [Jellyfin Forums](https://forum.jellyfin.org/) will depened on active forum members having experience with SCALE.
```
Comment (Gauvino) [inline docs/general/installation/truenas-scale.md:13]:
```suggestion
  - Seeking help for TrueNAS SCALE on the [Jellyfin Forums](https://forum.jellyfin.org/) will depends on active forum members having experience with SCALE.
```

File: docs/general/installation/truenas-scale.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+8. In the main Apps page, you will be able to use the installed Jellyfin app to launch it in a new browser by clicking on the `Web Portal` button. You can also simply open Jellyfin by using the port number assigned during its creation in a browser (ex: your-truenas-ip:port#).
+
+---
+
+## Custom App Install Steps
+
+TrueNAS SCALE also provides access to deploying custom apps not provided by the community charts. You can use this to deploy a Jellyfin app entirely controlled by you.
+
+1. Follow steps 1 & 2 for the previous community app section.
```
Comment (Gauvino) [inline docs/general/installation/truenas-scale.md:51]:
```suggestion
1. Follow steps 1 & 2 from the previous community app section.
```
Like this maybe ?

**Review by Gauvino — CHANGES_REQUESTED** [review]

**Comment by felix920506** [comment]
I would like to kindly ask for the truenas scale guide to be separated out into a different PR so that we can merge the Truecharts part first then slowly work on the truenas scale guide.

File: docs/general/installation/truenas-scale.md (hunk 2 above)
Comment (maru801) [reply docs/general/installation/truenas-scale.md:13]: It probably would be better to change this to "depends on".

File: docs/general/installation/truenas-scale.md (hunk 3 above)
Comment (maru801) [reply docs/general/installation/truenas-scale.md:51]: 👍

**Comment by maru801** [comment]
Closing this in favor of: https://github.com/jellyfin/jellyfin.org/pull/1163

**Pull request closed without merging**